### PR TITLE
Initial commit of solution pack

### PR DESCRIPTION
**Pull request by dcspille**

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.4.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "SOC Utilities",
+    "dependencies": [],
+    "prerequisite": "",
+    "publisher": "Fortinet CSE",
+    "description": "This Solution pack is designed around utilities to aid in the use of SOAR",
+    "help": null,
```
Comment (cs-suyog-jinturkar) [inline info.json]: We need documentation created for this SP and its respective readme.md path should be mentioned here as well

File: info.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "sOCUtilities",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.4.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "SOC Utilities",
+    "dependencies": [],
```
Comment (cs-suyog-jinturkar) [inline info.json]: You might want to depend upon SFSP

 {
            "name": "sOARFramework",
            "type": "solutionpack",
            "label": "SOAR Framework",
            "minVersion": null
        }

File: info.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "prerequisite": "",
+    "publisher": "Fortinet CSE",
+    "description": "This Solution pack is designed around utilities to aid in the use of SOAR",
```
Comment (cs-suyog-jinturkar) [inline info.json:18]: Also we need to add below key;
"certified": "false",

File: info.json (hunk 1 above)
Comment (cs-suyog-jinturkar) [reply info.json]: e.g. "help": "https://github.com/fortinet-fortisoar/solution-pack-typosquat-alert-investigation/blob/develop/README.md",

**Comment by cs-suyog-jinturkar** [comment]
We also built a documentation generator for SPs. We shall share with you once its available on public github. Stay tune

File: info.json (hunk 1 above)
Comment (cs-suyog-jinturkar) [reply info.json]: Documentation can be created by following steps mentioned here: https://github.com/fortinet-fortisoar/how-tos/blob/main/documenting/README.md

File: info.json (hunk 3 above)
Comment (dcspille) [reply info.json:18]: This has been fixed

File: info.json (hunk 1 above)
Comment (dcspille) [reply info.json]: Added the documentation in Readme, contents, setup, and usuage md files.

**Review by dcspille — COMMENTED** [review]
Finished reviewing requests

File: info.json (hunk 2 above)
Comment (dcspille) [reply info.json]: This has been added

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**